packages/services/pkg/faucet/core.go

```diff
@@ -47,13 +48,23 @@ func VerifySig(from, sigHex string, msg []byte) (bool, string) {
 	return from == recoveredAddr.Hex(), recoveredAddr.Hex()
 }
 
-func ExtractSignatureFromTweet(tweet twitter.Tweet) string {
-	tokens := strings.Split(tweet.FullText, "faucet")
-	return strings.TrimSpace(tokens[1])[:132]
+func ExtractSignatureFromTweet(tweet twitter.Tweet) (string, error) {
+	// TODO: decide on format of tweet to recover sig.
+	in := strings.NewReader(tweet.FullText)
```

Comment:
can we find the emoji string (prob any continuous string of emojis of length > 5) from the tweet? i can see us adding more text to the tweet.